database/config.go

```diff
@@ -0,0 +1,100 @@
+package database
+
+import (
+	"time"
+
+	"github.com/flyteorg/flytestdlib/config"
+	"gorm.io/gorm/logger"
+)
+
+const (
+	database = "database"
+	postgres = "postgres"
+)
+
+//go:generate pflags DbConfig --default-var=defaultConfig
+
+var defaultConfig = &DbConfig{
+	MaxIdleConnections: 10,
+	MaxOpenConnections: 1000,
+	ConnMaxLifeTime:    config.Duration{Duration: time.Hour},
+	Postgres: &PostgresConfig{
+		Port:         5432,
+		User:         postgres,
+		Host:         postgres,
+		DbName:       postgres,
+		ExtraOptions: "sslmode=disable",
+	},
+}
+var configSection = config.MustRegisterSection(database, defaultConfig)
+
+// DbConfig is used to for initiating the database connection with the store that holds registered
+// entities (e.g. workflows, tasks, launch plans...)
+type DbConfig struct {
+	DeprecatedHost         string `json:"host" pflag:",deprecated"`
+	DeprecatedPort         int    `json:"port" pflag:",deprecated"`
+	DeprecatedDbName       string `json:"dbname" pflag:",deprecated"`
+	DeprecatedUser         string `json:"username" pflag:",deprecated"`
+	DeprecatedPassword     string `json:"password" pflag:",deprecated"`
+	DeprecatedPasswordPath string `json:"passwordPath" pflag:",deprecated"`
+	DeprecatedExtraOptions string `json:"options" pflag:",deprecated"`
+	DeprecatedDebug        bool   `json:"debug" pflag:",deprecated"`
```

Comment:
```suggestion
        // deprecated: Please use Postgres.Host or SQLLite.Host instead
	DeprecatedHost         string `json:"host" pflag:"-,deprecated"`
	
	// deprecated:... <same for all properties>
	DeprecatedPort         int    `json:"port" pflag:"-,deprecated"`
	DeprecatedDbName       string `json:"dbname" pflag:"-,deprecated"`
	DeprecatedUser         string `json:"username" pflag:"-,deprecated"`
	DeprecatedPassword     string `json:"password" pflag:"-,deprecated"`
	DeprecatedPasswordPath string `json:"passwordPath" pflag:"-,deprecated"`
	DeprecatedExtraOptions string `json:"options" pflag:"-,deprecated"`
	DeprecatedDebug        bool   `json:"debug" pflag:"-,deprecated"`
```

Reply:
updated it, I didn't add SQLLite.Host in the comment because we only have one property `File` in `SQLiteConfig`